package.json

```diff
@@ -1,44 +1,34 @@
 {
   "name": "@bigcommerce/stencil-utils",
   "version": "0.4.0",
-  "main": "dist/stencil-utils.min.js",
+  "main": "src/main.js",
```

Comment:
Talked offline, the dist file still exists at dist/stencil-utils.min.js, but the main now points to the es6 source code for standard module imports.

dist bundle can still be dropped in as a `<script>` tag on the markup.